src/Microsoft.Diagnostics.DebugServices.Implementation/Runtime.cs

```diff
@@ -62,10 +62,14 @@ void IDisposable.Dispose()
             _serviceContainer.RemoveService(typeof(IRuntime));
             _serviceContainer.DisposeServices();
             _onFlushEvent.Dispose();
+            Flush();
```

Comment:
Should this go to the beginning of the method?

Reply:
Moved it.